vercel_runtime/src/http.rs

```diff
@@ -0,0 +1,96 @@
+use lambda_http::{http::StatusCode, Body, Error, Response};
+use serde::Serialize;
+use std::error::Error as StdError;
+use tracing::error;
+
+pub fn ok<T: Serialize>(val: T) -> Result<Response<Body>, Error> {
+    Ok(Response::builder()
+        .status(StatusCode::OK)
+        .header("content-type", "application/json")
```

Comment:
I wonder if it's specifically worth having these as `json_http` or something, because it all sets the content-type 🤔. 

Food for thought.

Reply:
If we always serialize as json then we can leave it :think: